docs/tutorials/moving-pictures/moving-pictures.md

```diff
@@ -0,0 +1,336 @@
+
+If you've completed the DEICODE tutorial already, you can follow along here
+(using the output files you produced from DEICODE).
+If you haven't completed the DEICODE tutorial already, you can download the
+necessary files for this tutorial here:
+
+- `table.qza` [view](https://view.qiime2.org/?src=https%3A%2F%2Fdocs.qiime2.org%2F2019.4%2Fdata%2Ftutorials%2Fmoving-pictures%2Ftable.qza) | [download](https://docs.qiime2.org/2019.4/data/tutorials/moving-pictures/table.qza)
+- `sample-metadata.tsv` [download](https://data.qiime2.org/2019.4/tutorials/moving-pictures/sample_metadata.tsv)
+- `taxonomy.qza` [view](https://view.qiime2.org/?src=https%3A%2F%2Fdocs.qiime2.org%2F2019.4%2Fdata%2Ftutorials%2Fmoving-pictures%2Ftaxonomy.qza) | [download](https://docs.qiime2.org/2019.4/data/tutorials/moving-pictures/taxonomy.qza)
+- `ordination.qza` [view](https://view.qiime2.org/?src=http%3A%2F%2Fbiocore.github.io%2Fqurro%2Ftutorials%2Fmoving-pictures%2Fdata%2Fordination.qza) | [download](http://biocore.github.io/qurro/tutorials/moving-pictures/data/ordination.qza)
+- `biplot.qzv` [view](https://view.qiime2.org/?src=http%3A%2F%2Fbiocore.github.io%2Fqurro%2Ftutorials%2Fmoving-pictures%2Fdata%2Fbiplot.qzv) | [download](http://biocore.github.io/qurro/tutorials/moving-pictures/data/biplot.qzv)
+
+## Running Qurro
+
+Since we'll be working with DEICODE output (i.e. feature loadings in a
+compositional biplot), we'll need to use the
+`qiime qurro unsupervised-rank-plot` command. (The `supervised-rank-plot`
+command is for working with feature differentials.)
+
+```
+qiime qurro unsupervised-rank-plot \
+    --i-table table.qza \
+    --i-ranks ordination.qza \
+    --m-sample-metadata-file sample-metadata.tsv \
+    --m-feature-metadata-file taxonomy.qza \
+    --o-visualization qurro-plot.qzv
+```
+
+##### Output Artifacts
+
+- `qurro-plot.qzv` [view](https://view.qiime2.org/?src=http%3A%2F%2Fbiocore.github.io%2Fqurro%2Ftutorials%2Fmoving-pictures%2Fdata%2Fqurro-plot.qzv) | [download](http://biocore.github.io/qurro/tutorials/moving-pictures/data/qurro-plot.qzv)
+
+You just generated your first Qurro plot! `qurro-plot.qzv` is a `.qzv` file --
+that is, a QIIME 2 visualization. You can view it either by running `qiime tools
+view qurro-plot.qzv` or by uploading the file to
+[view.qiime2.org](https://view.qiime2.org/). (You can also just click on the
+"view" link right above to see a precomputed version of this visualization.)
+
+## Interacting with a Qurro visualization
+
+Let's view `qurro-plot.qzv`, as described above.
+
+<img src="https://raw.githubusercontent.com/biocore/qurro/master/docs/tutorials/moving-pictures/screenshots/qurro1.png" alt="Qurro interface screenshot #1. No features are selected." />
+
+So right away we see two things: on the left a plot of rankings (in this case,
+loadings) for each feature, and on the right a plot of selected features' log
+ratios in samples. In this tutorial these plots will be referred to as the
```

Comment:
ass comma after `In this tutorial`

Reply:
Not sure I agree with that. In any case it now says `Throughout this tutorial these plots ...` -- sound ok?